src/appkit.cs

```diff
@@ -19704,10 +20142,32 @@ public interface NSSharingServiceDelegate
 		
 		[Export ("sharingService:sourceWindowForShareItems:sharingContentScope:"), DelegateName ("NSSharingServiceSourceWindowForShareItems"), DefaultValue (null)]
 		NSWindow SourceWindowForShareItems (NSSharingService sharingService, NSObject [] items, NSSharingContentScope sharingContentScope);
+	
+		[Export ("anchoringViewForSharingService:showRelativeToRect:preferredEdge:"), DelegateName ("NSSharingServiceAnchoringViewForSharingService"), DefaultValue (null)]
+		[return: NullAllowed]
+		NSView AnchoringViewForSharingService (NSSharingService sharingService, ref CGRect positioningRect, ref NSRectEdge preferredEdge);
```

Comment:
Also drop `ForSharingService` in method's name as this is inferred by first param.